bootstrap4/.js/src/main/scala/io/udash/bootstrap/datepicker/UdashDatePicker.scala

```diff
@@ -57,46 +89,17 @@ final class UdashDatePicker private[datepicker](
   def disable(): Unit =
     jQInput.datetimepicker("disable")
 
-  val render: Element = {
-    jQInput.datetimepicker(optionsToJsDict(options.get))
-
-    propertyListeners += options.listen { opts =>
-      optionsToJsDict(opts).foreach { case (key, value) =>
-        jQInput.datetimepicker(key, value)
-      }
-    }
-
-    date.get.foreach(d => jQInput.datetimepicker("date", dateToMoment(d)))
-    propertyListeners += date.listen(op => op.foreach(d => jQInput.datetimepicker("date", dateToMoment(d))))
-
-    nestedInterceptor(new JQueryOnBinding(jQInput, "change.datetimepicker", (_: Element, ev: JQueryEvent) => {
-      val event = ev.asInstanceOf[DatePickerChangeJQEvent]
-      val dateOption = event.option.flatMap(ev => sanitizeDate(ev.date)).map(momentToDate)
-      val oldDateOption = date.get
-      if (dateOption != oldDateOption) {
-        dateOption match {
-          case Some(null) => date.set(None)
-          case Some(d) => date.set(Option(d))
-          case None => date.set(None)
-        }
-        fire(UdashDatePicker.DatePickerEvent.Change(this, dateOption, oldDateOption))
-      }
-    }))
-    nestedInterceptor(new JQueryOnBinding(jQInput, "hide.datetimepicker", (_: Element, ev: JQueryEvent) => {
-      fire(UdashDatePicker.DatePickerEvent.Hide(this, date.get))
-    }))
-    nestedInterceptor(new JQueryOnBinding(jQInput, "show.datetimepicker", (_: Element, ev: JQueryEvent) => {
-      fire(UdashDatePicker.DatePickerEvent.Show(this))
-    }))
-    nestedInterceptor(new JQueryOnBinding(jQInput, "error.datetimepicker", (_: Element, ev: JQueryEvent) => {
-      fire(UdashDatePicker.DatePickerEvent.Error(this, date.get))
-    }))
-
+  override val render: Element = {
+    propertyListeners += options.listen(opts =>
+      optionsToJsDict(opts).foreach { case (optionKey, optionValue) => jQInput.datetimepicker(optionKey, optionValue) }
+    )
+    propertyListeners += date.listen(optionalDate => optionalDate.foreach(date => jQInput.datetimepicker("date", dateToMoment(date))))
     inp
   }
 
   override def kill(): Unit = {
     super.kill()
```

Comment:
Usual order is to first kill your stuff, then parent/super. This is due to the fact that `super` is initialized before `this` and some of your stuff might depend on it, hence the stack cleanup order.